### PR TITLE
add feature to execute user-defined command when repair is successfully finished

### PR DESCRIPTION
**Pull request by YZ775**
In order to implement Non-Graceful Node Shutdwon, this PR adds feature to execute user-defined command when repair is successfully finished.

### CODE REVIEW

File: docs/repair.md [hunk 1]
```diff
@@ -108,6 +108,12 @@ CKE decides to execute a repair operations if its `operation` matches `OPERATION
 When CKE executes the check command, it appends the IP address of the target machine to the command.
 The command should return a string `true` if it evaluates the machine as healthy.
 
+`success_command` and its timeout are used when all of the repair steps are successfully finished.
```
Comment (morimoto-cybozu) [inline docs/repair.md:111]: This description seems misleading.
CKE in this PR will execute `success_command` when it finds that `health_check_command` returns `true`.
* CKE will abort the sequence of the repair steps if the machine gets healthy; not all of the repair steps are executed.
* Even when all of the repair steps finish successfully, the machine may still be unhealthy.

```suggestion
`success_command` and its timeout are used when the machine is evaluated as healthy and the repair operation finishes successfully.
```

File: docs/repair.md [hunk 2]
```diff
@@ -108,6 +108,12 @@ CKE decides to execute a repair operations if its `operation` matches `OPERATION
...
+When CKE executes the success command, it appends the IP address of the target machine to the command.
+If the repair is failed, the command is not executed.
```
Comment (morimoto-cybozu) [inline docs/repair.md:113]:
```suggestion
If the repair operation has failed, the command is not executed.
```

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]
This feature seems to work perfectly!

I added comments just for clarity.

**Comment by YZ775** [comment]
@morimoto-cybozu
Thank you for your review. 
I fixed the code and document.

**Review by morimoto-cybozu — APPROVED** [review]

**Pull request merged**